task/history_collector.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright (c) 2015 VMware, Inc. All Rights Reserved.
+Copyright (c) 2015, 2022 VMware, Inc. All Rights Reserved.
 
```

Comment:
The header linter expects a range, changing to `2015, 2022` to `2015-2022` will work

Reply:
Updated. Thank you.